src/test/java/elizCurtnazar/tests/tema7/MakeAppointmentScenariosTest.java

```diff
@@ -0,0 +1,50 @@
+package elizCurtnazar.tests.tema7;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.List;
+
+public class MakeAppointmentScenariosTest extends BaseTest{
+
+    @Test
+    public void clickMakeAppointmentButton() {
```

Comment:
You are missing a negative scenario for appointment. From what I can see this is only the positive scenario. Please double check the homework requirement. Thanks!

Reply:
ok